pyravendb/store/entity_to_json.py

```diff
@@ -0,0 +1,69 @@
+import pyravendb.tools.utils
+from pyravendb import constants
+from copy import deepcopy
+import json
+
+
+class EntityToJson:
+    def __init__(self, session):
+        self._session = session
+        self._missing_dictionary = dict()
+
+    @property
+    def missing_dictionary(self):
+        return self._missing_dictionary
+
+    def convert_entity_to_json(self, entity, document_info):
+        if document_info is not None:
+            self._session.events.before_conversion_to_json_internal(document_info["key"], entity)
+        document = EntityToJson._convert_entity_to_json_internal(self, entity, document_info)
+        if document_info is not None:
+            self._session.events.after_conversion_to_document(document_info["key"], entity, document)
+        return document
+
+    def _convert_entity_to_json_internal(self, entity, document_info, remove_identity_property=False):
+        json_node = json.loads(json.dumps(entity, default=self._session.conventions.json_default_method))
```

Comment:
don't you have a util wrapping dumps + loads already?

Reply:
replaced with util